driver-core/src/main/java/com/datastax/driver/core/DataType.java

```diff
@@ -188,7 +188,7 @@ static DataType decode(ByteBuf buffer, ProtocolVersion protocolVersion, CodecReg
                     DataType fieldType = decode(buffer, protocolVersion, codecRegistry);
                     fields.add(new UserType.Field(fieldName, fieldType));
                 }
-                return new UserType(keyspace, type, fields, protocolVersion, codecRegistry);
+                return new UserType(keyspace, type, true, fields, protocolVersion, codecRegistry);
```

Comment:
If the default frozenness is `false`, why are we setting it to `true` when reading resultset metadata? I know it doesn't matter much in this context, but for consistency I wonder if we shouldn't consider all user types as non-frozen when reading metadata from a result response.